mappings/net/minecraft/block/BlockShape.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_3581 net/minecraft/block/BlockShape
```

Comment:
VoxelShape is the upstream name

Reply:
This is different, I guess `SideShapeType` is the closest in newer versions

Reply:
ah ok. i personally think ``BlockShape`` is better than ``SideShapeType`` so leave it that way